user/pages/03.commerce2/02.developer-guide/02.install-update/01.requirements/docs.md

```diff
@@ -0,0 +1,38 @@
+---
+title: Requirements
+taxonomy:
+    category: docs
+---
+
+ ### Requirements
+
+ -Commerce 2.x requires Drupal 8.5.0 or newer.
+
+ Unless you are building a very simple site, you will normally want to develop
+ locally, on a copy of your site that operates separately from the live site.
+
+ If you already have a web server, make sure it satisfies [Drupal 8’s requirements].
+ The recommended memory limit is 256MB or more. For local development we recommend
+ [DDEV] (Docker-based), [Drupal VM] (advanced users), or [Acquia Dev Desktop] (beginners). You will also need [Composer].
```

Comment:
I don't think we should call DrupalVM advanced.. we should just say it's Vagrant based. So we're offering a Docker solution, Vagrant.. and then "custom".

Reply:
Also, Acquia Dev Desktop got abandoned in the meantime, so it's not safe to recommend it anymore.